res/code_templates/relationship-right/custom/Extension/modules/__module__/Ext/Vardefs/idx_for___relationship-right__.php.twig

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * @author SugarCli
+ * SugarCli created file
+ */
+
+// Relationship foreign key to {{ moduleLeft|replace({'_': ' '}) }} index
+$dictionary["{{ moduleBean }}"]["indices"][] = array(
```

Comment:
In Sugar 7.7 I don't see length as an available option for a index. Regardless, I see it as being a manually set similar to change varchar length for fields.